### ISSUE TITLE
Mini-Monterey-Patcher 0.1.0: get busy, guys!!

### ISSUE THREAD

**Issue by shines75** [Ursinia/Mini-Monterey-Patcher#13]
I would premise that in my opinion, Mini-Monterey-Patcher, for now, as of 7/10/21, come to version 0.1.0, is a promising patcher for the coming Monterey, but certainly a lot of work has still to be done.
Why? I'm Italian, and this early morning, as soon as version 0.1.0 was released, enthusiastic I remade the Monterey USB stick, patched it with the new version, reinstalled Monterey beta 2 on a dedicated external ssd, and tried to patch it with the new PatchSystem.sh, (I've a MacBook Pro mid-2012), but...alas, here my pains began: patching the system from within Monterey, seems to give almost any results...something seems that it's done on disk, but in fact I got absolutely nothing, apart from a malfunctioning wi-fi! On the other hand, patching the system from the Monterey USB stick gave me the wi-fi and the graphics acceleration, but no audio at all (!) and, oh well, no bluetooth.
But the worst was yet to come: despite the lack of audio and the bluetooth, I tried to migrate my data from my external ssd with Big Sur 11.4 installed and patched with Patched-Sur, with Migration Assistant, and all seemed to work good until the end, when the app asked me to restart the mac to complete the migration. Rather quiet, I did it and, after the mac restarted and began to boot from Monterey, after 6-7 seconds...It restarted and crashed!!! That was too much...for sure, I know  I don't have severe incompatible software installed on my external ssd, because some time ago I give a try to OpenCore-Legacy Patcher, (btw, wonderful patcher, pity the graphics acceleration sucks, at least on my machine), and tried to install the first beta of Monterey on the dedicated external ssd mentioned before, patching it with OpenCore-Legacy and repeating the procedure of migration described above...and result? All run terribly slow, because the graphics acceleration suck, as said before, but everything went fine!!
All this, to say that then there are two cases: either this issue is due for a very strange reason, just to Monterey beta 2, and then to Apple, or maybe the issue is due to your patcher...and sincerely, I'd lean towards the second hypothesis.
In short, definitely we aren't there yet, Ben & co.

**Comment by BenSova** [comment]
> with Migration Assistant, and all seemed to work good until the end, when the app asked me to restart the mac to complete the migration. Rather quiet, I did it and, after the mac restarted and began to boot from Monterey, after 6-7 seconds...It restarted and crashed!!!

That's an Apple bug for sure. Migration Assistant sucks in my experience.

> I know I don't have severe incompatible software installed on my external ssd, because some time ago I give a try to OpenCore-Legacy Patcher

OCLP + Mini Monterey = Explosion.

> patching the system from within Monterey, seems to give almost any results

Logs?

>  lack of audio

Afaik that hardware works natively. I can check again, but I don't remember seeing that was a problem.

**Comment by Monkiey** [comment]
> I would premise that in my opinion, Mini-Monterey-Patcher, for now, as of 7/10/21, come to version 0.1.0, is a promising patcher for the coming Monterey, but certainly a lot of work has still to be done.
> Why? I'm Italian, and this early morning, as soon as version 0.1.0 was released, enthusiastic I remade the Monterey USB stick, patched it with the new version, reinstalled Monterey beta 2 on a dedicated external ssd, and tried to patch it with the new PatchSystem.sh, (I've a MacBook Pro mid-2012), but...alas, here my pains began: patching the system from within Monterey, seems to give almost any results...something seems that it's done on disk, but in fact I got absolutely nothing, apart from a malfunctioning wi-fi! On the other hand, patching the system from the Monterey USB stick gave me the wi-fi and the graphics acceleration, but no audio at all (!) and, oh well, no bluetooth.
> But the worst was yet to come: despite the lack of audio and the bluetooth, I tried to migrate my data from my external ssd with Big Sur 11.4 installed and patched with Patched-Sur, with Migration Assistant, and all seemed to work good until the end, when the app asked me to restart the mac to complete the migration. Rather quiet, I did it and, after the mac restarted and began to boot from Monterey, after 6-7 seconds...It restarted and crashed!!! That was too much...for sure, I know I don't have severe incompatible software installed on my external ssd, because some time ago I give a try to OpenCore-Legacy Patcher, (btw, wonderful patcher, pity the graphics acceleration sucks, at least on my machine), and tried to install the first beta of Monterey on the dedicated external ssd mentioned before, patching it with OpenCore-Legacy and repeating the procedure of migration described above...and result? All run terribly slow, because the graphics acceleration suck, as said before, but everything went fine!!
> All this, to say that then there are two cases: either this issue is due for a very strange reason, just to Monterey beta 2, and then to Apple, or maybe the issue is due to your patcher...and sincerely, I'd lean towards the second hypothesis.
> In short, definitely we aren't there yet, Ben & co.

if Monterey run slow keep in mind its a beta os also after upgrading file indexing takes place give it time

**Comment by shines75** [comment]
I’ve not done OCLP + Mini Monterey Patcher. Moreover, see that audio really lacks in Monterey patched with your MMP!!!

**Comment by shines75** [comment]
And to Monkiey: you misrepresented completely what I said in the post…I said that Monterey patched with OCLP runs terribly slow!!!

**Comment by Monkiey** [comment]
> And to Monkiey: you misrepresented completely what I said in the post…I said that Monterey patched with OCLP runs terribly slow!!!

yeah because its a beta os and cut your attitude its really rude and demanding

**Comment by ozziepadilla** [comment]
> And to Monkiey: you misrepresented completely what I said in the post…I said that Monterey patched with OCLP runs terribly slow!!!

Sorry but you don’t even know what are you saying

**Comment by Monkiey** [comment]
these devs have done so much for the community and people like you ruin it as devs get upset with the toxicity you crate and leave the community

**Comment by BenSova** [comment]
I can't confirm that, since it works perfectly fine. When hardware has it get patched it's because it doesn't work universally since it's the same hardware. If it works on my MBP which is the exact same as yours, it should work fine for you too, something else must be blocking it.

**Comment by Monkiey** [comment]
there is a reason why I use big sur right now  because Monterey is still in development and is not finalized if you want your mac to work don't use betas

**Comment by ozziepadilla** [comment]
> > And to Monkiey: you misrepresented completely what I said in the post…I said that Monterey patched with OCLP runs terribly slow!!!
> 
> yeah because its a beta os and cut your attitude its really rude and demanding

Yeah I am agree, you did a mess with your Mac in your long comment you said it , stop being rude and show respect ,
Lacks of audio ? You are wring I did a test on my mac , everything works as Ben said Bluetooth won’t work and the continuity too

**Comment by BenSova** [comment]
> And to Monkiey: you misrepresented completely what I said in the post…I said that Monterey patched with OCLP runs terribly slow!!!

You probably didn't apply the post install patches.

**Comment by Monkiey** [comment]
> You probably didn't apply the post install patches.

Yeah that can definitely be the issue

**Comment by Monkiey** [comment]
that is why people should read instructions before  reporting issue have the issue are because people don't read instcutions

**Comment by ozziepadilla** [comment]
> there is a reason why I use big sur right now because Monterey is still in development and is not finalized if you want your mac to work don't use betas

Me too , if don’t understand English let me say all this in Italian 

macOS Monterey è ancora in versione beta ed è molto lento oltre al mucchio di bug, stai cercando di fare qualcosa che incasina OCLP e Patchet-Sur che non funzionerà

**Comment by ozziepadilla** [comment]
> that is why people should read instructions before reporting issue most people have issues are because people don't read instructions

I am agree Ben is right , read the instructions first before use , or may be keep away from Childrens in your case stay away if you don’t know what are u doing

**Comment by BenSova** [comment]
Let's pretend like that didn't happen.

**Comment by Monkiey** [comment]
> Let's pretend like that didn't happen.

yes

**Comment by Monkiey** [comment]
> Let's pretend like that didn't happen. Sorry shines75.

yesh sorry to you @shines75 i will think next time with more logic

**Comment by shines75** [comment]
E va bene, caro ozziepadilla, te lo dico in italiano, visto che mi avete riempito di insulti (dandomi dell’illogico, del tossico, etc…): l’inglese credo di conoscerlo e di capirlo bene, (sul tuo italiano invece avrei un po’ da ridire…), e a parte quindi sentirmi profondamente offeso, credo che non posterò più niente su Github…voi sviluppatori siete un po’ troppo permalosi e pretendete di avere sempre ragione. E ora, se volete, continuate pure a dirmi che creo tossicità nel vostro ambiente d’elitè.

**Comment by shines75** [comment]
And anyway, it’s a kinder garden!

**Comment by Monkiey** [comment]
> And anyway, it’s a kinder garden!

i am sorry that you have to witness this behavior please don't one bad apple ruin how feal about this community

**Comment by Monkiey** [comment]
my sincere and deepest apologies to you @shines75 what you have witnessed here was ben trying to help you and Ozzie and let me honest me adding fuel to the fire once ben told us to stop Ozzie continued this behavior while I listened and have learned from it.

**Comment by shines75** [comment]
Okay, okay, I might as well be a bad apple but you have a really special jargon, a bit incomprehensible to the laymen like me.  Over and out!

**Comment by Monkiey** [comment]
yes the behavior was unacceptable so apologies for that, hope you have a good day

**Comment by AvaQueen-Official** [comment]
Thank you all for the apologies, once again we are very sorry for this @shines75. I hope you take our apology professionally. I have discussed this matter with the others and they have agreed to improve. Do you have any further questions regarding the initial issue?

**Comment by shines75** [comment]
But do you at least realize that yours is a street Englishman?  What the f****..

**Comment by shines75** [comment]
Scusate lo sfogo. Noi italiani siamo fatti così, quando vediamo le cose storte!!!

**Comment by shines75** [comment]
Sorry for the outburst.  We Italians are made in this way, when we see things wrong!!!

**Comment by Monkiey** [comment]
> Sorry for the outburst. We Italians are made in this way, when we see things wrong!!!

i understand this behavior on here from some was still professional so you had the right to be in a state of outburst, i hope you understand ou apologies and everyone involved can move on, Have a wonderful day

**Comment by BenSova** [comment]
@shines75 sorry about their behavior, hopefully we can get past that and figure out where your bug is coming from.

 Also, regarding #10 and #11, were those solved by v0.1.0?

**Comment by shines75** [comment]
Thank you, Ben Sova. I appreciate very much kindness and correct language. I think now we begin to reason…however, I believe that before I was pretty straightforward: in the end, I’ve not understood the reason why audio lacks, (I believed it was a patching issue, but I was wrong), and above all, why Migration Assistant throws a tantrum. Apart that, however, issues #10 and #11 are solved, and I’ll close them later. Thanks for reminding them to me.

**Comment by Monkiey** [comment]
I am glad your issues got resolved

**Comment by AvaQueen-Official** [comment]
thank you for staying with us despite the setbacks some of the team caused you

**Comment by shines75** [comment]
Um, let's smooth it out out divergences…we are humans, after all.

**Comment by shines75** [comment]
Our divergences: sorry for the typo

**Comment by Monkiey** [comment]
> Our divergences: sorry for the typo

its okay we all make typos

**Comment by shines75** [comment]
Ben Sova? Are you still here?

**Comment by BenSova** [comment]
Yes

**Comment by shines75** [comment]
What do you think? How are things really? Maybe if you try to explain them again to me using a very correct english…

**Comment by shines75** [comment]
Meanwhile, I’ll close the 2 previous issues for correctness.

**Comment by shines75** [comment]
I do apologise, but I don’t know how to close an issue. Might someone kindly explain me how to do that?

**Comment by AvaQueen-Official** [comment]
@shines75 if you opened an issue, there should be a "close" button or "close with comment" at the bottom of the page. If you didn't open the issue, there will not be such button.

**Comment by shines75** [comment]
Okay, let’s see

**Comment by shines75** [comment]
I don’t know. I’m with my iPad Pro and I don’t see nothing on the bottom of the pages of my issues.

**Comment by BenSova** [comment]
I'll take care of it

**Comment by BenSova** [comment]
Done

**Comment by shines75** [comment]
What have you done?

**Comment by BenSova** [comment]
Closed the issues

**Comment by shines75** [comment]
Ah, perfect. Thank you very much

**Comment by BenSova** [comment]
> What do you think? How are things really? Maybe if you try to explain them again to me using a very correct english…

So I'm not sure why audio isn't working. It could be a Monterey bug and it could also be third party software interfering, but the hardware is still supported.

Regarding your Time Machine backup not working, that's definitely a Monterey bug, not much I can do there.

**Comment by shines75** [comment]
Yes, but I hope you do realize you’re saying that Apple is guilty of both

**Comment by AvaQueen-Official** [comment]
@shines75 There is a reason why this is a beta. Naturally, some things might not work, especially on an unsupported machine. Nothing wrong or incorrect with saying that.

**Comment by shines75** [comment]
Aaah, so the real reason might be the fact I’m running the beta on my unsupported machine

**Comment by AvaQueen-Official** [comment]
> Aaah, so the real reason might be the fact I’m running the beta on my unsupported machine

Exactly, or that you're running the beta at all 👍🏻

**Comment by shines75** [comment]
Yes, so the reasoning runs, but I have to contradict you: when, for experiment, I tried to patch Monterey beta 1 with OpenCore Legacy, everything went fine…what ruined all was the fact that on my machine it ran terribly slow, due to the fact graphics acceleration in OCP sucks (I suppose on my machine only).  And then? How do we put the things now?

**Comment by AvaQueen-Official** [comment]
@shines75 that is not a contradiction - unsupported macs require patches to run properly, and you either didn't do the post-install patches on OCLP or something happened while you were doing them that they didn't patch properly.

**Comment by shines75** [comment]
Excuse me, I was talking with Ben

**Comment by AvaQueen-Official** [comment]
> Excuse me, I was talking with Ben

you need to make yourself clear who you're talking to - this is a oneline thread

**Comment by Monkiey** [comment]
> Excuse me, I was talking with Ben

excuse me that attitude will get you nowhere please listen to others advice as well

**Comment by shines75** [comment]
Pardon me, I was distracted

**Comment by BenSova** [comment]
> Yes, so the reasoning runs, but I have to contradict you: when, for experiment, I tried to patch Monterey beta 1 with OpenCore Legacy, everything went fine…what ruined all was the fact that on my machine it ran terribly slow, due to the fact graphics acceleration in OCP sucks (I suppose on my machine only). And then? How do we put the things now?

Just asked a dev of OCLP, they have no patches for audio on your Mac because "AppleHDA didn't drop any layout IDs".

**Comment by BenSova** [comment]
So audio is fully supported, OCLP or Mini Monterey. The reason why it doesn't work is probably something on Apple's side.

**Comment by shines75** [comment]
Eh, Ben Sova, but audio worked on monterey beta 1 patched with OCLP…instead beta 2 patched with your MMP, not! Is it clear, now?

**Comment by BenSova** [comment]
I know. Maybe if you tried beta 1 it would work, who knows. But I did confirm with the developer of OCLP and he confirmed that your audio is supported without any extra patches.

**Comment by Monkiey** [comment]
> Eh, Ben Sova, but audio worked on monterey beta 1 patched with OCLP…instead beta 2 patched with your MMP, not! Is it clear, now?

![Screen Shot 2021-07-10 at 11 12 07 AM](https://user-images.githubusercontent.com/28370827/125172650-b50bc900-e16f-11eb-9b8c-e7726daa348a.png)

**Comment by shines75** [comment]
> So audio is fully supported, OCLP or Mini Monterey. The reason why it doesn't work is probably something on Apple's side.

Yes, notwithstanding that on your machine, that is equal to mine, beta 2 works with the audio

**Comment by BenSova** [comment]
That's how a lot of bugs are. Someone on iOS 15 can't see their Memojis while a lot of others can. It's part of how bugs are, they're conditional.

**Comment by Monkiey** [comment]
> That's how a lot of bugs are. Someone on iOS 15 can't see their Memojis while a lot of others can. It's part of how bugs are, they're conditional.

yeah an liek you siad "The most common problems only occur for a specific scenario and won’t occur for any other scenario. "

**Comment by shines75** [comment]
Mmmm, it seems to be a real puzzle. I give it up. I’ll wait for both the official releases of Monterey and Monterey Patcher. The life is already complicated enough.
